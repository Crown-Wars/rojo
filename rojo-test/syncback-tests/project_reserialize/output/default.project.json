{
  "name": "project_reserialize",
  "tree": {
    "$className": "DataModel",
    "Workspace": {
      "attribute_mismatch": {
        "$attributes": {
          "foo": "bar"
        },
        "$path": "attribute_mismatch.luau"
      },
      "property_mismatch": {
        "$path": "property_mismatch.project.json"
      },
      "$properties": {
        "CSGAsyncDynamicCollision": {
          "Enum": 0
        },
        "DecreaseMinimumPartDensityMode": {
          "Enum": 0
        },
        "StreamingEnabled": true
      },
      "$attributes": {
        "Rojo_Target_CurrentCamera": "6d6ae1d713c82fae0620aa1300000375"
      }
    }
  }
}