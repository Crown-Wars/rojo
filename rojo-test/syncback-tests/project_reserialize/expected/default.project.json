{
  "name": "project_reserialize",
  "tree": {
    "$className": "DataModel",
    "Workspace": {
      "$properties": {
        "CSGAsyncDynamicCollision": {
          "Enum": 0
        },
        "DecreaseMinimumPartDensityMode": {
          "Enum": 0
        },
        "StreamingEnabled": true
      },
      "$attributes": {
        "Rojo_Target_CurrentCamera": "6d6ae1d713c82fae0620aa1300000375"
      }
    }
  }
}